app/js/event/event.filter.js

```diff
@@ -1,59 +0,0 @@
-(function() {
-    'use strict';
-
-    angular
-        .module('upcomingStops.event')
-        .factory('eventFilter', eventFilter);
-
-    eventFilter.$inject = ['$http'];
-
-    // function eventFilter($http) {
```

Comment:
If this is a factory I think everything should be wrapped inside this function (after renaming it to eventSort)